pkg/config/filter.go

```diff
@@ -12,7 +12,12 @@ import (
 	"github.com/open-policy-agent/opa/bundle"
 )
 
-func FilterIgnoredPaths(paths, ignore []string, checkFileExists bool) ([]string, error) {
+func FilterIgnoredPaths(paths, ignore []string, checkFileExists bool, rootDir string) ([]string, error) {
```

Comment:
This is a breaking change, I guess, but hopefully not one anyone has come to rely on 😬 And if they did, it should be easy to fix.

Reply:
Discussed on slack, after being unable to find any public integrations based on this we have opted to merge and leave a note in the release.